source/style-guide/seo-guidelines.txt

```diff
@@ -7,7 +7,48 @@ Search Engine Optimization Guidelines
 Search Engine Optimization (SEO) considers the actual terms that users 
 enter into search engines (keywords) and employs best practices to 
 improve traffic to web pages from search engines. The following 
-guidance should help you optimize your documentation pages for search.
+guidance helps you to optimize your documentation pages for search.
+
+.. important::
+
+   In addition to this guidance, our content taxonomy improves 
+   findability and increases relevance for search queries. To learn 
+   more about the instructions and best practices for our content 
+   taxonomy, see the `Taxonomy tagging instructions 
+   <https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions>`__. 
+
+Keywords
+--------
+
+You can add the actual terms that users enter into search engines 
+(keywords) to the body of your page to improve its SEO.
+
+.. important::
+  
+   This guidance makes a distinction between the terms "keywords" and 
+   "keyword meta tags". Here "keywords" are popular search terms added 
+   throughout the content body, and "keyword meta tags" are metadata 
```

Comment:
Question:
I think I am confused by this description. I originally thought this was talking about SEO keywords which are described here as "keyword meta tags" given that the other section is about the "meta description".

I think "[add keywords] to the body of your page" might be too general and the example didn't help clarify where the keyword could be added.

Suggestion:
If I understand correctly, this advice is to add/use the "keywords" throughout the page copy. For example, if you know the terms that people search for to try to find this page are "MongoDB Atlas Course", you should add that phrase in at least one spot. This could be the example:

Without keyword phrase:
"Take the MongoDB University "Getting Started With Atlas" course."

With keyword phrase:
"Learn more by taking the **MongoDB Atlas Course** called "Getting Started With Atlas"."

If I'm still missing something, please reach out to discuss on Slack or Zoom!